pagopa/io-app
# chore: [IAI-100] Add generic test overlay

## Short description
This pr adds a new generic test overlay, used in internal beta test in order to mark a version as **not** release candidate. The message could be changed using the env variable `TEST_OVERLAY_CAPTION`.

<img width="300" alt="Schermata 2021-12-02 alle 15 20 09" src="https://user-images.githubusercontent.com/26501317/144439769-1806953d-9265-4572-a00c-1f7d998e6e4f.png">

## List of changes proposed in this pull request
- Removed `BPD_TEST_OVERLAY`
- Added `TEST_OVERLAY_CAPTION`
- Minor graphical adjustment in `ts/components/BetaTestingOverlay.tsx`

## How to test
- Enable in `.env` file the `TEST_OVERLAY_CAPTION`
- Run the app
- See the overlay
